hazelcast/src/main/java/com/hazelcast/core/IMap.java

```diff
@@ -1803,13 +1815,13 @@ boolean tryLock(K key, long time, TimeUnit timeunit, long leaseTime, TimeUnit le
     ICompletableFuture submitToKey(K key, EntryProcessor entryProcessor);
 
     /**
-     * Applies the user defined EntryProcessor to the all entries in the map.
+     * Applies the user defined {@link EntryProcessor} to the all entries in the map.
      * Returns the results mapped by each key in the map.
      */
     Map<K, Object> executeOnEntries(EntryProcessor entryProcessor);
 
     /**
-     * Applies the user defined EntryProcessor to the entries in the map which satisfies provided predicate.
+     * Applies the user defined {@link EntryProcessor} to the entries in the map which satisfy provided predicate.
```

Comment:
...which satisfy the...

Reply:
"satisfy" referes to the "entries", not to the "map" i think

Reply:
Right. I was thinking it was missing the `the` article before `provided` but I can see it works without it as well.